files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -232,6 +232,7 @@
     path: /etc/ssh/sshd_config
     regexp: "^#?{{ item.key }} "
     line: "{{ item.key }} {{ item.value }}"
+  notify: "sshd config changed"
```

Comment:
Minor nit: it looks like the convention throughout this file is to place the notify section at the end of the task. I don't think it practically matters, but just to follow the existing stylistic pattern...

Reply:
Sure seb, updated.